data/src/test/java/org/apache/iceberg/io/TestRollingFileWriters.java

```diff
@@ -39,15 +38,15 @@
 @RunWith(Parameterized.class)
 public abstract class TestRollingFileWriters<T> extends WriterTestBase<T> {
 
-  // TODO: add ORC once we support ORC rolling file writers
-
   @Parameterized.Parameters(name = "FileFormat={0}, Partitioned={1}")
   public static Object[] parameters() {
     return new Object[][] {
         new Object[]{FileFormat.AVRO, false},
         new Object[]{FileFormat.AVRO, true},
         new Object[]{FileFormat.PARQUET, false},
         new Object[]{FileFormat.PARQUET, true},
+        new Object[]{FileFormat.ORC, false},
+        new Object[]{FileFormat.ORC, true},
```

Comment:
I found another test class `TestBaseTaskWriter` also need to enable the ORC parameter.